apple/metal_library.bzl

```diff
@@ -106,6 +115,12 @@ metal_library = rule(
             allow_files = METAL_FILE_TYPES,
             doc = """\
 A list of `.metal` source files that will be compiled into the library.
+""",
+        ),
+        "hdrs": attr.label_list(
```

Comment:
Isn't it necessary to make `hdrs` mandatory?

Reply:
Yes. Some metal file need import header file. Issue #39 is an example.